tests/foreman/installer/test_installer.py

```diff
@@ -1586,3 +1586,108 @@ def test_installer_timestamp_logs():
 
     :CaseAutomation: NotAutomated
     """
+
+
+@pytest.mark.stubbed
+@pytest.mark.tier3
+def test_positive_capsule_installer_and_register():
+    """Verify the capsule installation and their registration with the satellite.
+
+    :id: efd03442-5a08-445d-b257-e4d346084379
+
+    :steps:
+        1. Install the satellite.
+        2. Add all the required cdn and custom repositories in satellite to
+           install the capsule.
+        3. Create life-cycle environment,content view and activation key.
+        4. Subscribe the capsule with created activation key.
+        5. Run 'yum update -y' on capsule.
+        6. Run 'yum install -y satellite-capsule' on capsule.
+        7. Create a certificate on satellite for new installed capsule.
+        8. Copy capsule certificate from satellite to capsule.
+        9. Run the satellite-installer(copy the satellite-installer command from step7'th
+            generated output) command on capsule to integrate the capsule with satellite.
+        10. Check the newly added capsule is reflected in the satellite or not.
+        11. Check the capsule sync.
+
+
+    :expectedresults:
+
+        1. Capsule integrate successfully with satellite.
+        2. Capsule sync should be worked properly.
+
+    :CaseImportance: Critical
```

Comment:
looks like 12 tests are critical, 80 % we can mark that at file level and update the 2 tests with high. (non blocking )